client/docs/package-template/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "package-name",
+  "license": "agpl-3.0-or-later",
+  "description": "Short package description.",
+  "version": "1.0.0",
+  "keywords": [
+    "wikijump"
+  ],
+  "scripts": {
+    "clean": "trash dist .tsbuildinfo",
+    "build": "node ../../build.js -self"
+  },
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scpwiki/wikijump.git"
+  },
+  "bugs": {
+    "url": "https://github.com/scpwiki/wikijump/issues"
```

Comment:
Issues are disabled, the link for requests/bugs from the public is https://scuttle.atlassian.net/servicedesk/customer/portal/2

Reply:
Resolved via [this commit.](https://github.com/scpwiki/wikijump/pull/181/commits/3d9e77348d620dfc75f3bc09bc840a80eeb47f81)